pkg/processor/runtime/runtime.go

```diff
@@ -60,6 +62,15 @@ type Runtime interface {
 
 	// SupportsRestart return true if the runtime supports restart
 	SupportsRestart() bool
+
+	// SupportsControlCommunication
+	SupportsControlCommunication() bool
+
+	// ConsumeControlMessage returns a channel that receives control messages
+	ConsumeControlMessage() <-chan *controlcommunication.ControlMessage
```

Comment:
```suggestion
	ConsumeControlMessages() <-chan *controlcommunication.ControlMessage
```

Reply:
Also, consume/subscribe functionality is not runtime per-se interface, but "controlMessage" one, perhaps, the runtime, can expose a "GetControlMessageBroker" (or so) which holds that interface (and more), and use that instead